src/coreComponents/physicsSolvers/multiphysics/PoromechanicsSolver.hpp

```diff
@@ -118,6 +114,10 @@ class PoromechanicsSolver : public CoupledSolver< FLOW_SOLVER, MECHANICS_SOLVER
                    GEOS_FMT( "{} {}: The attribute `{}` of the flow solver `{}` must be set to 1 since the poromechanics solver is thermal",
                              this->getCatalogName(), this->getName(), FlowSolverBase::viewKeyStruct::isThermalString(), this->flowSolver()->getName() ),
                    InputError );
+
+    DomainPartition & domain = this->template getGroupByPath< DomainPartition >( "/Problem/domain" );
+    flowSolver()->initialize( domain );
```

Comment:
need to call this first to initialize porosity and densities

Reply:
it's a bunch of things including hydrostatic equilibrium
multiphase equilibration is not yet supported, see https://github.com/GEOS-DEV/GEOS/blob/e601fad71502e0af6b0117af2f8c212ee11ca720/src/coreComponents/physicsSolvers/fluidFlow/CompositionalMultiphaseBase.cpp#L1153